libbeat/docs/command-reference.asciidoc

```diff
@@ -910,6 +910,13 @@ messages.
 *`-e, --e`*::
 Logs to stderr and disables syslog/file output.
 
+*`-environment`*::
+Tell {beatname_uc} the environment it runs into. If no log output is configured
```

Comment:
It's not clear up front that this setting relates to logging. Here's my suggested edit based on what I think you're saying here:

```
For logging purposes, specifies the environment that {beatname_uc} is running in.
This setting is used to select a default log output when no log output is configured.
Supported values are: `systemd`, `container`, `macos_service`, and `windows_service`.
If `systemd` or `container` is specified, {beatname_uc} will log to stdout and stderr
by default.
```

Reply:
Thanks, will update. Initially I didn't want to this setting to be about logging only. A generic flag like `environment` is not really bound to logging. Other subsystems could also be configured differently based on the environment. On the other hand I'd prefer to not have this flag at all...